hw/ip/otbn/doc/_index.md

```diff
@@ -630,10 +630,93 @@ However, every error that OTBN detects when it isn't running is fatal.
 This means that the cause will be reflected in {{< regref "FATAL_ALERT_CAUSE" >}}, as described below in [Alerts]({{< relref "#alerts" >}}).
 This way, no alert is generated without setting an error code somewhere.
 
+### List of Errors {#design-details-list-of-errors}
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>BAD_DATA_ADDR<code></td>
+      <td>recoverable</td>
+      <td>A data memory access occurred with an out of bounds or unaligned access.</td>
+    </tr>
+    <tr>
+      <td><code>BAD_INSN_ADDR<code></td>
+      <td>recoverable</td>
+      <td>An instruction memory access occurred with an out of bounds or unaligned access.</td>
+    </tr>
+    <tr>
+      <td><code>CALL_STACK<code></td>
+      <td>recoverable</td>
+      <td>An instruction tried to pop from an empty call stack or push to a full call stack.</td>
+    </tr>
+    <tr>
+      <td><code>ILLEGAL_INSN<code></td>
+      <td>recoverable</td>
+      <td>
+        An illegal instruction was about to be executed.
+        An instruction is illegal in the following cases:
+        <ul>
+          <li>An instruction being excuted had an invalid encoding.</li>
+          <li>An access occurred for an invalid CSR or WSR.</li>
+        </ul>
+      </td>
+    <tr>
+      <td><code>LOOP<code></td>
+      <td>recoverable</td>
+      <td>
+        A loop stack-related error was detected.
+        One of the following things happened:
+        <ul>
+          <li>A loop was started with an iteration count of zero.</li>
+          <li>The final instruction of a loop was a branch or another loop.</li>
+          <li>A new loop tried to push to a full loop stack (loop nesting level too deep).</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>FATAL_IMEM<code></td>
```

Comment:
Another topic: I'd probably remove the "FATAL_" prefix from the error names. We now have this information in the table itself. And with the upcoming feature to promote recoverable errors to fatal errors, having the prefix seems more confusing than helping.

Reply:
Is it just a namespacing thing to avoid clashes in the RTL? If not, I'm happy to drop it.

Reply:
No issues with dropping it